src/bindings/fluid_cmd.c

```diff
@@ -1000,6 +1024,28 @@ fluid_handle_echo(void* data, int ac, char** av, fluid_ostream_t out)
   return FLUID_OK;
 }
 
+/* Purpose:
+ * Sleep during a time in ms
+ * The command itself is useful to insert a delay between commands.
+ * It can help for exemple to build a small song using noteon/noteoff commands
+ * in a command file.
+ */
+int
+fluid_handle_sleep(void *data, int ac, char** av, fluid_ostream_t out)
+{
+  if (ac < 1) {
+    fluid_ostream_printf(out, "sleep: too few arguments.\n");
+    return -1;
+  }
+  if (!fluid_is_number(av[0])) {
+    fluid_ostream_printf(out, "sleep: argument should be a number in ms.\n");
+    return -1;
+  }
+  g_usleep(atoi(av[0]) * 1000);	/* delay in micro second.*/	
```

Comment:
This should now be `fluid_msleep(atoi(av[0]))`

Reply:
Yes,it will be done.